classes/class-wc-rest-connect-address-normalization-controller.php

```diff
@@ -35,14 +35,13 @@ public function post( $request ) {
 			return $error;
 		}
 
-		if ( isset( $response->error ) ) {
-			$error = new WP_Error(
-				$response->error->code,
-				$response->error->message,
-				array( 'message' => $response->error->message )
+		if ( isset( $response->fieldErrors ) ) {
+			$response = array(
+				'success' => true,
+				'fieldErrors' => $response->fieldErrors,
 			);
-			$this->logger->debug( $error, __CLASS__ );
-			return $error;
+			$this->logger->debug( $response, __CLASS__ );
```

Comment:
This causes a warning: `PHP Notice:  Array to string conversion in /srv/www/wordpress-default/public_html/wp-content/plugins/woocommerce-services/classes/class-wc-connect-logger.php on line 39`